src/types/plugin-input.ts

```diff
@@ -13,6 +13,10 @@ export interface PluginInputs<T extends SupportedEventsU = SupportedEventsU, TU
 
 const ONE_DAY = 24 * 60 * 60 * 1000;
 
+const userRoleSchema = T.Union([T.Literal("Admin"), T.Literal("Member"), T.Literal("Contributor")]);
```

Comment:
I think this might prohibit role extension since this is a literal union of three items when it could potentially be something else